src/improvements/justfile

```
@@ -116,13 +116,21 @@ simulate-all-templates:
     forge build
 
     # If the task is nested then we only simulate as the foundation.
-    # In the future we could simulate as other nested safes. 
+    # In the future we could simulate as other nested safes.
     # For testing purposes, we do not gain anything by simulating as other nested safes.
     nested_safe_name="foundation"
+    gen_verify_input="true"
+
     simulation_count=0
     for task in ${root_dir}/test/tasks/example/*/*; do
         if [ -d "$task" ]; then
-            ${root_dir}/src/improvements/script/simulate-task.sh $task $nested_safe_name
+            current_dir=$(pwd)
+            # Derive a filepath for the verify input. e.g if the task is in src/improvements/tasks/example/eth/000-gas-config-template.sol
+            # then the verify input will be in current_dir/OPVerifyInputs/eth/000-gas-config-template-input.json in the current directory
+            op_verify_input_filepath="${current_dir}/OPVerifyInputs/$(basename "$(dirname "${task}")")/${task##*/}-input.json"
+            # Create the directory for the verify input if it doesn't exist
```

```suggestion
            # Create the directory for the verify input if it does not exist
```

This apostrophe is messing with my justfile plugin 🤔